src/PKSim.Core/Snapshots/Mappers/CompoundMapper.cs

```diff
@@ -43,7 +43,7 @@ public class CompoundMapper : ParameterContainerSnapshotMapperBase<ModelCompound
          snapshot.Permeability = await mapAlternatives(compound, COMPOUND_PERMEABILITY);
          snapshot.PkaTypes = mapPkaTypes(compound);
          snapshot.Processes = await mapProcesses(compound);
-         snapshot.IsSmallMolecule = SnapshotValueFor(compound.IsSmallMolecule, true);
+         snapshot.IsSmallMolecule = compound.IsSmallMolecule;
```

Comment:
This was indeed not saved for compound (Based on the original principal that default value should not be saved because it's just noise in the file)